python/lsst/ts/standardscripts/auxtel/mock/attcs_mock.py

```diff
@@ -0,0 +1,296 @@
+            )
+
+            if self.track:
+                self.atmcs.evt_target.set_put(elevation=self.tel_alt,
+                                              azimuth=self.tel_az,
+                                              force_output=True)
+
+            await asyncio.sleep(1.)
+
+    async def dome_tel_pos_telemetry(self):
+        while self.run_telemetry_loop:
+            self.atdome.tel_position.set_put(azimuthPosition=self.dom_az)
+            await asyncio.sleep(1.)
+
+    async def atmcs_wait_and_fault(self, wait_time):
+        self.atmcs.evt_summaryState.set_put(summaryState=salobj.State.ENABLED,
+                                            force_output=True)
+        await asyncio.sleep(wait_time)
+        self.atmcs.evt_summaryState.set_put(summaryState=salobj.State.FAULT,
+                                            force_output=True)
+
+    async def atptg_wait_and_fault(self, wait_time):
+        self.atptg.evt_summaryState.set_put(summaryState=salobj.State.ENABLED,
+                                            force_output=True)
+        await asyncio.sleep(wait_time)
+        self.atptg.evt_summaryState.set_put(summaryState=salobj.State.FAULT,
+                                            force_output=True)
+
+    async def fake_slew_callback(self, id_data):
+        """Fake slew waits 5 seconds, then reports all axes
+           in position. Does not simulate the actual slew.
+        """
+        self.atmcs.evt_allAxesInPosition.set_put(inPosition=False,
+                                                 force_output=True)
+        self.atdome.evt_azimuthInPosition.set_put(inPosition=False,
+                                                  force_output=True)
+        self.track = True
+        self.task_list.append(asyncio.create_task(self.wait_and_send_inposition()))
+
+    async def fake_move_dome(self, data):
+        self.atdome.evt_azimuthInPosition.set_put(inPosition=False,
+                                                  force_output=True)
+
+        await asyncio.sleep(self.slew_time)
+
+        self.atdome.tel_position.set(azimuthPosition=data.azimuth)
+        self.atdome.evt_azimuthInPosition.set_put(inPosition=True,
+                                                  force_output=True)
+
+    async def fake_stop_tracking(self, data):
+        pass
+
+    async def wait_and_send_inposition(self):
+
+        await asyncio.sleep(self.slew_time)
+        self.atmcs.evt_allAxesInPosition.set_put(inPosition=True,
+                                                 force_output=True)
+        await asyncio.sleep(0.5)
+        self.atdome.evt_azimuthInPosition.set_put(inPosition=True,
+                                                  force_output=True)
+
+    async def generic_callback(self, id_data):
+        await asyncio.sleep(0.5)
+
+    async def move_shutter_callback(self, id_data):
+        # This command returns right away in the current version of the dome.
```

Comment:
No, it doesn't. v0.8.0 waits for the motion to finish.

Reply:
I wrote it before v0.8.0... :)

Reply:
Can you update the script for the current behavior?